api/users/views.py

```diff
@@ -18,11 +21,16 @@ class UserMixin(object):
 
     def get_user(self, check_permissions=True):
         key = self.kwargs[self.node_lookup_url_kwarg]
+        current_user = self.request.user
 
         if key == 'me':
-            return self.request.user
+            if isinstance(current_user, AnonymousUser):
+                raise PermissionDenied
```

Comment:
i just updated the PR description, but the reason this is a 403 currently is because https://github.com/CenterForOpenScience/osf.io/pull/3821 has yet to be completed. once that is finished, they will be changed to 401's